Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.7
+
+ARG SRC=app
+
+COPY requirements.txt /opt/app/requirements.txt
+WORKDIR /opt/app
+RUN pip install -r requirements.txt
+
+RUN groupadd prod && useradd -g prod prod
+USER prod:prod
+
+COPY ${SRC} app
+COPY ${SRC}/../templates templates
```

Comment:
remove this line from the docker file, that way the test frontend wont be available in production